### PR TITLE
Add text to the flags to avoid problems with non OS X users

### LINKED ISSUE #12: Flags on demo page does not appear in a non OS X computer.
### Description
When you open the demo, you don't see the flags.

### Expected outcome

You should see the english and french flags.

### Actual outcome

You see a characters.

app-localize-behavior-translate

### Steps to reproduce

+ Open the demo in a non OS X  computer. For example, Ubuntu.

### PR DESCRIPTION
**Pull request by felixzapata**
Fixes #12

### CODE REVIEW

File: demo/x-local-translate.html [hunk 1]
```diff
@@ -18,9 +18,9 @@
     <style include="common-styles"></style>
 
     <div class="lang">
-      <span title="english">🇬🇧</span>
+      <span title="english">🇬🇧 (english)</span>
```
Comment (notwaldorf) [inline demo/x-local-translate.html]: Can we just have `🇬🇧  EN - FR 🇫🇷`, please? I think it looks a little bit more cleaner/symmetric.

**Comment by notwaldorf** [comment]
Good point! I've just left a small comment, before I can merge this PR. Thanks!

**Comment by felixzapata** [comment]
done @notwaldorf

**Comment by notwaldorf** [comment]
Thanks so much! 👍

**Comment by felixzapata** [comment]
you´re welcome

**Pull request merged**